content/administration/upgrade/on_premise.rst

```diff
@@ -37,13 +37,30 @@ Upgrade request via the Odoo Upgrade Portal
    - Any problem found during testing should be reported via the `helpdesk
      <https://odoo.com/help>`_.
 
+.. note::
+   - For storage reasons, the copy of your database is submitted without a filestore to the upgrade server, therefore,
+     the upgraded database does not contain the production filestore.
+   - Before restoring the upgraded database, its filestore must be merged with the production filestore to be able to perform
+     tests in the same conditions as it would be in the new version.
+   
```

Comment:
```suggestion

```
remove the spaces, as it leads to a `ci/documentation_guidelines` error:
![chrome_3flnwNLGTS](https://github.com/odoo/documentation/assets/70884563/b6d30b9d-8d72-46a1-892c-df3abb7d04da)

Reply:
I have a concern regarding the whitespaces
When I test locally, if I remove the spaces, the text is displayed in **bold**
![image](https://github.com/odoo/documentation/assets/28193899/1287a94d-2739-4d1a-bff7-f0daa27a22c0)
Is it the same behaviour when deployed in the Odoo doc?

Reply:
That's because a blank line (containing no character) is still necessary between the main bullet list and its "sub-list".
Sorry for not being clear enough! (should be fixed now)

Reply:
Thanks for clarifying that, I was not sure what was happening that made that bold :grin: